### ISSUE TITLE
Windows 10 v1809 Network Connectivity Status Indicator

### ISSUE THREAD

**Issue by ribatamu** [crazy-max/WindowsSpyBlocker#128]
There is a issue with the www.msftncsi.com and no connectivity internet tray icon on Windows 10 v1809. Previously I used another Windows build and I didn't had this problem.

I applied the custom NCSI probe and when I test it I have internet but my internet icon is with the no internet message status.

The only way to fix this icon is when I remove the entry www.msftncsi.com from the host file.

**Comment by crazy-max** [comment]
@ribatamu I was able to reproduce your issue. I will take a look, thanks for your input.

**Comment by xcom169** [comment]
> net but m

Do you have this with Spying or with Extra IP package?

**Comment by ribatamu** [comment]
@crazy-max I am still facing the issue with the version 4.18.4

**Comment by crazy-max** [comment]
@ribatamu Yes, I haven't worked on this problem yet. I'll keep you informed as soon as I have something.

**Comment by crazy-max** [comment]
@ribatamu Everything works as expected using Microsoft or WindowsSpyBlocker NCSI probes on clean Windows 10 (+ WindowsSpyBlocker hosts/firewall spy rules). Maybe you have something wrong with your [NCSI config](https://docs.microsoft.com/en-US/windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services#bkmk-ncsi) or a third party app blocks probes.

**Comment by llyama** [comment]
On a clean install of the latest Windows 10 LTSC (1809) NCSI is reporting no internet access when using the WindowsSpyBlocker NCSI probes. Doing a troubleshoot it reportedly thinks I am behind a hotspot, which I infer to mean the dns lookup worked but the web content failed. The test from within WindowSpyBlocker is successful.

I noticed both of the Microsoft test urls do not use https and raw.githubusercontent.com is https only. I found a pastebin type site that allows unsecured http and created a page with just "WindowsSpyBlocker " on it (http://termbin.com/d7y0) and updated the registry keys to use that. Windows reported internet access within a few minutes.

Another Windows 10 machine on 1709 continues to work with the WindowsSpyBlocker rules, so I'm guessing something changed within Windows.
